src/Domain/CampaignRepository.php

```diff
@@ -110,10 +117,16 @@ private function pullCharity(
 
         $tbgCanClaimGiftAid = (
             !empty($hmrcReferenceNumber) &&
-            in_array($giftAidOnboardingStatus, static::$giftAidOnboardedStatuses, true)
+            in_array($giftAidOnboardingStatus, self::GIFT_AID_ONBOARDED_STATUSES, true)
+        );
+        $tbgApprovedToClaimGiftAid = (
+            !empty($hmrcReferenceNumber) &&
+            in_array($giftAidOnboardingStatus, self::GIFT_AID_APPROVED_STATUSES, true)
```

Comment:
Am I right to think this relies on the changes about to go into SF, and therefore this PR shouldn't be merged to develop until after the SF one is in production?

Reply:
@bdsl In https://github.com/thebiggive/matchbot/pull/603/commits/8d247a3750a4e0d2b8de48104709da23163e889b I was thinking that setting a default would make it safe either way round, but I think you're right that there's a risk of subsequent data pulls putting the value back to a not-approved one. I don't think it's super critical though, at worst it might delay some claims slightly.

Reply:
Right, was thinking of it like a new field when matchbot might crash if the input doesn't have it, but as its just a new value I guess it should be safe enough, they'll show up as approved in matchbot the first time pull after the SF change is live.